src/db/tables/onerep_scans.ts

```diff
@@ -78,6 +81,7 @@ async function addOnerepScanResults(
         onerep_profile_id: onerepProfileId,
         onerep_scan_id: onerepScanId,
         onerep_scan_reason: onerepScanReason,
+        onerep_scan_status: onerepScanStatus,
```

Comment:
When running the scan locally or on dev we would not update the status with the polling method in order to workaround not having the webhooks available in `src/app/api/v1/user/welcome-scan/progress/route.ts`. In order to update the entry `onerep_scan_status` in the table `onerep_scan` we’ll need to update the status after `L77` I think and add something along the lines of:

```
await transaction("onerep_scans")
  .where("onerep_scan_id", onerepScanId)
  .update({
    onerep_scan_status: onerepScanStatus,
    updated_at: knex.fn.now(),
  });
```

Does that look correct @mansaj?

Reply:
Oh hmm... We have `getLatestOnerepScanResults`, which just gets the latest entry, so we could make it an append-only column, which I think was Joe's idea. But I also just realised that I made the `onerep_scan_id` unique, so if it should be append-only, I should probably update that as well.

Reply:
@Vinnl Ok I think I understand the issue here. In order for us to keep `onerep_scan_id` unique, we cannot make the table append only? Because two scan results would share the same scan ids?

If that's the case.. I think we'd probably still prefer append only for the `scan results` table (thus `onerep_scan_id` is no longer unique). I think it would be a lot easier for insert / retrieval, and our `getLatest` function already gets the latest. We also maintain a better record of the history of changes (if we need that later for audit purpose)... There are some of the pros I can think of

Reply:
@mansaj 

> In the `onerep_scans` table, the scan ids are still unique though right?

Not if it's append-only, because we need to know the latest status of a scan. So if it changes from `in_progress` to `complete`, we'd need to create a new row.

Reply:
Thinking out loud here.. I guess I'm wondering if it's silly idea to have `onerep_scans` be unique/update, and `onerep_scan_results` be append only?

Reply:
Yeah that works for me. I'll keep it as unique then :) Anyone want to approve the PR? :P

Reply:
@Vinnl Merged #3470 in here.